README.md

```diff
@@ -53,14 +53,23 @@ Die angeforderten Daten werden ebenfalls als JSON übermittelt.
 
 ## Authentifizierung
 
-Für jeden Request ist eine Authentifizierung erforderlich. Die Authentifizierung erfolgt über einen HTTP Header.
+Für jeden Request ist eine Authentifizierung erforderlich. Die Authentifizierung erfolgt über den OAuth 2.0 Client-Credentials Flow. 
```

Comment:
Ich fänd nett, wenn wir an dieser Stelle oder zumindest in https://github.com/europace/authorization-api an eine Beschreibung des Flows verlinken würden. Falls wir einen eigenen Artikel haben, dann natürlich den, ansonsten würde ich solche vorschlagen:
- näher an der Praxis: https://auth0.com/docs/flows/concepts/client-credentials
- der nüchterne Standard: https://tools.ietf.org/html/rfc6749#section-4.4

Reply:
ich würde das in der authorization-api sehen 🤔

Reply:
In der Auth-API ist der RFC verlinkt.